src/ecs/Menu.h

```diff
@@ -5,6 +5,7 @@
 #include <memory>
 #include <vector>
 #include <functional>
+#include <string>
```

Comment:
```
implicit instantiation of undefined template 'std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
```
on line 42
```
    std::string text;
```

This might be a macos gcc/clang issue?